src/battle_util.c

```diff
@@ -1660,6 +1661,10 @@ u8 DoBattlerEndTurnEffects(void)
             }
             gBattleStruct->turnEffectsTracker++;
             break;
+        case ENDTURN_FLINCH:  // reset flinch
+            gBattleMons[gActiveBattler].status2 &= ~(STATUS2_FLINCHED);
+            gBattleStruct->turnEffectsTracker++;
+            break;
```

Comment:
Since there are no battlescripts called, you can remove the break; statement, so the code executes faster.

Reply:
I put in the `break` for consistency with all the other cases - gone now :)